go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/codecentric/terraform-provider-leanix
+
+go 1.15
+
+require (
+	github.com/hashicorp/hcl/v2 v2.6.0
+	github.com/hashicorp/terraform v0.13.4
```

Comment:
Ist das hier ein sneaky Terraform upgrade? :)

Geht das einfach so, ohne, dass man den Providercode anpassen muss?

Reply:
Bei mir kompiliert's und die Unit Tests laufen auch durch! Die Acceptance Tests hab ich ned ausprobiert, weil ich keine LeanIX Keys hab :S

Allerdings weiß ich nicht ob das eine Mindestversion ist und der Provider dann nicht mehr mit alten Terraform-Versionen funktioniert 😅

Reply:
Ja super! Dann sollten wir noch einmal gegen eine LeanIX-Instanz testen bei Gelegenheit und wenn das klappt, dann können wir mergen und releasen. Wenn das nicht mehr mit alten Terraformversionen funktionieren sollte, wäre das auch nicht so schlimm. Dafür kann man ja dann die alte Version des Providers nehmen.

Ich bin nur verwundert weil ich dachte, dass sich irgendwas in der Paketstruktur geändert hat aber wer weiß. Hab es nicht mehr genau im Kopf.